core/gesture.js

```diff
@@ -166,6 +152,20 @@ Blockly.Gesture = function(e, touchId) {
    * @private
    */
   this.flyout_ = null;
+
+  /**
+   * Boolean for sanity-checking that some code is only called once.
+   * @type {boolean}
+   * @private
+   */
+  this.calledUpdateIsDragging_ = false;
```

Comment:
If I trust my code at the end of this change, I'll remove it.  If I mostly trust my code but think it might be subtly broken, I'll leave the asserts in for a few months to help shake out bugs.  I don't expect developers using blockly to run into this, but I do expect that you and I will.